tests/issues/run.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
-SCRIPT_DIR=$(dirname "$0")
+# Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+# Use of this source code is governed by a license that can be found
+# in the LICENSE file.
+
+SCRIPT_DIR=$(dirname $(realpath "$0"))
 
 for d in $(ls -d "${SCRIPT_DIR}/gh-"*/); do
-    echo "Running test for the issue $d"
-    ${d}/run.sh "$*"
+    echo "## `date "+%Y-%m-%dT%H:%M:%S"` | INFO | Running test for the issue $d"
```

Comment:
Move out log functions into separate shell file under "shared", include it into "CommonFunctions.sh" and here, use normal `_log` function instead of raw `echo`.